requirements.txt

```diff
@@ -1 +1,3 @@
-slackclient==1.0.9
+-e git+https://github.com/ckan/ckan#egg=ckan
+-r https://raw.githubusercontent.com/ckan/ckan/master/requirements.txt
```

Comment:
This would need fixing/revert as well.

Reply:
Right. I don't quite understand what you mean here. The contents here are auto generated by pip complie. The second file contains the project requirements. Are you asking me to remove it?

Reply:
The `requirements.txt` would list out what's needed for the plugin to work that aren't listed as CKAN requirements already. The extension works within CKAN's environment so we would not have to install those once more.

You'll notice the kind of requirements that are listed for other extensions like the harvester - https://github.com/ckan/ckanext-harvest/blob/master/pip-requirements.txt

On fixing - this extension does need `slackclient==1.0.9` which has been removed in this PR and isn't part of CKAN's requirements. It would mean that Slack notifications wouldn't work. Make sense?